addon/services/apollo.ts

```diff
@@ -97,27 +112,30 @@ export default Service.extend({
    * @return {!Object}
    * @public
    */
-  clientOptions: computed(function() {
+  @computed
```

Comment:
Is there a need to make this a computed even if they are not dependent of any properties? Or should we add `link`,`cache` here?

```suggestion
  @computed('link', 'cache')
```

Reply:
It is `computed` to be cached.